AUTHORS.rst

```diff
@@ -3,7 +3,6 @@ Maintainers
 
 - Bryce Boe <bbzbryce@gmail.com> `@bboe <https://github.com/bboe>`_
 - Joe RH <jarhill0@gmail.com> `@jarhill0 <https://github.com/jarhill0>`_
-- Daniel P. <pyprohly@outlook.com> `@Pyprohly <https://github.com/Pyprohly>`_
```

Comment:
There’s no reason for me to be up here. I don’t need to be a Maintainer to make good contributions. I just don’t want the sense of responsibility right now. This doesn’t otherwise change anything.

Reply:
Makes sense.